roles/pulp-routes/defaults/main.yml

```diff
@@ -10,6 +10,12 @@ hostname: '{{ deployment_type }}.example.com'
 # https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size
 nginx_client_max_body_size: 10m
 
+nginx_proxy_read_timeout: 90s
+nginx_proxy_connect_timeout: 90s
+nginx_proxy_send_timeout: 90s
+
+haproxy_timeout: 120s
```

Comment:
@ipanova wdyt about these values?

Reply:
I would not know what would be best default values for the HAproxy, strictly depends how the pulp deployment will be used, but the logic is constructed this way, where client should have the biggest timeout.

client <--->HAProxy <---> nginx <---> gunicorn <---> wsgi app

We already have 90secs default timeout for the gunicorn meaning that nginx ones should be bigger.
Usually the rule is to add  5 seconds, I think, but for the readability sake we could go with:
client <--->HAProxy (180) <---> nginx(120) <---> gunicorn (90)<---> wsgi app


@bmbouter are these values acceptable, wdyt?

Reply:
Yup, it all sounds fine to me.